web/html/src/components/combobox.tsx

```diff
@@ -57,6 +57,14 @@ export class Combobox extends React.Component<ComboboxProps, ComboboxState> {
 
         return styles;
       },
+      menu: (styles: {}) => ({
+        ...styles,
+        zIndex: 3,
```

Comment:
This looks a bit arbitrary. Why is this 3, but not say 1 or 10? If the specific value is important it would be good to add a comment here outlining the rationale.

Reply:
well, I just copy/pasted it from the `Select` code... and the place where I saw the issue at that time I had to go over a `zIndex` of 2... modals are `1000` IIRC but it seems to work fine

Reply:
Fair enough, can't expect you to add information you don't know. 😄

Reply:
hehe :)